home_work_1/mains.py

```diff
@@ -0,0 +1,40 @@
+# print string "I love Python"
+
+count = 0
+
+while count < 42:
```

Comment:
The goal was not to use loops but to learn how to manipulate strings using arithmetical operators. Please, rewrite this part using multiplication (*)